backend/store.py

```
@@ -1224,6 +1398,132 @@ def clear_jobs(self):
+            job_id (str): Identifier stored as the lock owner when acquisition succeeds.
+            ttl_seconds (int): Time-to-live for the lock in seconds (defaults to 600).
+        
+        Returns:
+            `True` if the lock was acquired and the job_id was stored as owner, `False` otherwise.
+        """
+        key = self._cluster_lock_key(project_id)
+        if self.mode == "redis":
+            return bool(self.client.set(key, job_id, nx=True, ex=ttl_seconds))
+        try:
+            result = self.client.set_with_opts(key, job_id, "NX", "EX", str(ttl_seconds))
+        except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)